### ISSUE TITLE
Synchronize watched videos and their progress via iCloud

### ISSUE THREAD

**Issue by pdiazmon** [zagahr/Conferences.digital#14]
Sometimes I watch videos on my laptop and sometimes I do on my desktop Mac. Could be great if I have the same watched videos list and progress on both.

**Comment by zagahr** [comment]
Good idea. The only problem is that I would need an Apple Developer account in order to access CloudKit. I might consider buying a membership.
